e2e_playwright/test_assets/hostframe.html

```diff
@@ -187,21 +191,6 @@
             url: "https://rsms.me/inter/font-files/Inter-Bold.woff2?v=3.19",
             weight: 700,
           },
-          {
-            family: "Apercu Mono",
-            url: "https://app.snowflake.com/static/2c4863733dec5a69523e.woff2",
-            weight: 400,
-          },
-          {
-            family: "Apercu Mono",
-            url: "https://app.snowflake.com/static/e903ae189d31a97e231e.woff2",
-            weight: 500,
-          },
-          {
-            family: "Apercu Mono",
-            url: "https://app.snowflake.com/static/32447307374154c88bc0.woff2",
-            weight: 700,
-          },
```

Comment:
The fonts from https://app.snowflake.com/static/ cannot be requested because of CORS.